azext_iot/product/providers/auth.py

```diff
@@ -0,0 +1,58 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from msrest.authentication import Authentication
+
+
+class AICSAuthentication(Authentication):
```

Comment:
Note for later really (not expecting you to do in this PR) - This auth model along with DT / PnP / Central can be shared. We don't need to redefine.